localization/replace/english/wc_dlc_fp1_l_english.yml

```diff
@@ -17,18 +17,18 @@
  jomsviking_pirates_flavor:0 "#F Some of the most notorious reavers ever to ply the North Sea, the $jomsviking_pirates$ excel at injecting chaos into battle, drawing it out before excelling in the aftermath.#!"
  
- fp1_yearly_0071_troll_victory_bear_modifier_desc:0 "This character is reputed to have slain either a dire troll, or the world's largest bear."
- fp1_yearly_0071_troll_victory_wolf_modifier_desc:0 "This character slew a wolf so swift that the commonfolk believed it a dire troll."
- fp1_yearly_0071_troll_victory_wolverine_modifier_desc:0 "This character slew a wolverine so ferocious that the commonfolk believed it a dire troll, keeping the wilds safe."
- fp1_yearly_0071_troll_victory_lynx_modifier_desc:0 "This character slew a lynx so silent that the commonfolk believed it a dire troll."
- fp1_yearly_0071_troll_victory_moose_modifier_desc:0 "This character slew a moose so rabid that the commonfolk believed it a dire troll."
+ fp1_yearly_0071_troll_victory_bear_modifier_desc:0 "This character is reputed to have slain either a jalgar, or the world's largest bear."
+ fp1_yearly_0071_troll_victory_wolf_modifier_desc:0 "This character slew a wolf so swift that the commonfolk believed it a jalgar."
+ fp1_yearly_0071_troll_victory_wolverine_modifier_desc:0 "This character slew a wolverine so ferocious that the commonfolk believed it a jalgar, keeping the wilds safe."
+ fp1_yearly_0071_troll_victory_lynx_modifier_desc:0 "This character slew a lynx so silent that the commonfolk believed it a jalgar."
+ fp1_yearly_0071_troll_victory_moose_modifier_desc:0 "This character slew a moose so rabid that the commonfolk believed it a jalgar."
  
- fp1_yearly_0071_troll_attacks_modifier:0 "Dire Troll Attacks"
+ fp1_yearly_0071_troll_attacks_modifier:0 "Jalgar Attacks"
 
- fp1_yearly.0031.desc.hunted_troll:1 ""... 'course, this most recent wound is from the dire troll itself. Big as a house, quiet as a mouse, claws like the bastard spawn of [lying_warrior.GetFaith.DevilName]. Still, stab anything long enough, and it dies.""
- fp1_yearly.0071.desc.outro.loveli_lakes:0 "\n\nI have picked over the mess for only a short time when [loveli_lakes.GetTitledFirstName] steps forward.\n\n"This is no troll, [ROOT.Char.GetFirstNameNoTooltip], this is the work of a feral moose! I know the beast well, including its nasty bite, and would carve my mark on its corpse.""
- fp1_yearly.0071.desc.outro.fallback:0 "\n\nThe locals insist this is nothing less than the work of a dire troll, and demand I scour nearby woods to dispatch for the creature."
- fp1_yearly.0081.hunt_invalidated:0 "Hunt for the Dire Troll of [ROOT.Char.GetCapitalLocation.GetCounty.GetTitle.GetNameNoTierNoTooltip] Ceases"
+ fp1_yearly.0031.desc.hunted_troll:1 ""... 'course, this most recent wound is from the jalgar itself. Big as a house, quiet as a mouse, claws like the bastard spawn of [lying_warrior.GetFaith.DevilName]. Still, stab anything long enough, and it dies.""
```

Comment:
Is it normal that there is double " around the sentence ?

Reply:
I think that's meant as verbal speech there.

Reply:
Yes, it's the same in vanilla
AFAIK it's a part of event text with direct speech